pages/common/flac.md

```diff
@@ -0,0 +1,16 @@
+# flac
+
+> Encodes, decodes and tests flac files.
+
+
+- Encode a wav file to flac, writing to /path/to/file.flac:
+
+`flac {{/path/to/file.wav}}`
+
+- Decode a flac file to wav, specifying the output file:
```

Comment:
We should specify the `--output-name` flag with the encode first and then move to decode.